PRIVACY.md

```diff
@@ -0,0 +1,47 @@
+## Privacy Notice
+
+This privacy notice discloses the privacy practices for Flagrow Bazaar. This privacy notice applies solely to information collected by this application and its associated website at flagrow.io. It will notify of the following:
+
+* What personally identifiable information is collected from YOU through the website, how it is used and with whom it may be shared.
+* What choices are available to YOU regarding the use of your data.
+* The security procedures in place to protect the misuse of information YOU provide.
+* How YOU can correct any inaccuracies in the information.
+
+For the purposes of this notice, WE refers to the entity of Flagrow and its proprietors, YOU refers to a website that has installed Flagrow Bazaar and its owner, APPLICATION refers to Flagrow Bazaar, WEBSITE refers to flagrow.io. 
+
+### Information Collection, Use, and Sharing
+WE are the sole owners of the information collected by the APPLICATION during the course of normal operation. 
+
+If YOU choose to connect the APPLICATION to the WEBSITE, then WE also own any information provided. Unless YOU ask us not to, we may contact YOU via email in the future to tell you about specials, new products or services, or changes to this privacy policy.
+
+If YOU choose to email us or directly contact us in another way, WE also have access to/collect information that YOU voluntarily provide. WE will also use this information to respond to YOU, regarding the reason YOU contacted us. WE will not share your information with any third party outside of our organization, other than as necessary to fulfill your request, e.g. to provide a service.
+
+**WE will not sell or rent this information to anyone for any purposes.**
+
+### What We Collect
+
+To provide the best service for this APPLICATION, WE may collect the following from YOU. 
+
+* The contents of the Flarum configuration file where the APPLICATION is installed.
```

Comment:
This is far too wide. We do read the config, but we only use the url as mentioned 3 items down. I'd still leave this be, but mention something like "The contents of the Flarum config.php file, except for service authentication credentials" - feel free to modify.